JavaScript/JavaScriptSDK/TelemetryContext.ts

```diff
@@ -128,13 +130,20 @@ module Microsoft.ApplicationInsights {
             this._applyDeviceContext(envelope, this.device);
             this._applyInternalContext(envelope, this.internal);
             this._applyLocationContext(envelope, this.location);
-            this._applyOperationContext(envelope, this.operation);
             this._applySampleContext(envelope, this.sample);
             this._applyUserContext(envelope, this.user);
+            this._applyOperationContext(envelope, this.operation);
```

Comment:
Maybe I am missing something.. but why the change in order? If the order IS important, mention that in a comment?